ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.20__periodics.yaml

```diff
@@ -0,0 +1,96 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  ocp_4.20_base-rhel9:
+    name: "4.20"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-8-golang-1.23-openshift-4.19:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.23-openshift-4.19
+  ocp_builder_rhel-9-golang-1.23-openshift-4.19:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.23-openshift-4.19
+  upi-installer:
+    name: "4.20"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.23
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: ci
+      version: "4.20"
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.20"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-metal-ipi-ovn-dualstack-bgp-techpreview-periodic
+  cluster: build05
+  interval: 24h
```

Comment:
Due to limited metal resources during peak hours using cron running around 4 / 5 am utc will be better
 
cron: 0 4 * * *

Reply:
Also no need for periodic at the end, it is already coming in via the variant and currently shows 3 times :grin: 

periodic-ci-openshift-ovn-kubernetes-release-4.20-periodics-e2e-metal-ipi-ovn-dualstack-bgp-local-gw-techpreview-periodic

Reply:
ack, I will the rehearses advance enough to check they are ok and update.

Reply:
Regarding the naming, I didn't realize you already had these defined for [4.19](https://github.com/openshift/release/blob/87f399baf803d217014d4035eddc49c38d8a52b3/ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.19__periodics.yaml#L82) and earlier.  If you want to leave the trailing -periodic for consistency that would be fine.

Reply:
> Regarding the naming, I didn't realize you already had these defined for [4.19](https://github.com/openshift/release/blob/87f399baf803d217014d4035eddc49c38d8a52b3/ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.19__periodics.yaml#L82) and earlier. If you want to leave the trailing -periodic for consistency that would be fine.

I will remove it, too many periodics :P